examples/example_multivial.ipynb

```diff
@@ -0,0 +1,229 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from OptiHPLCHandler import EmpowerHandler\n",
+    "handler=EmpowerHandler(project=\"SLCB_2023_DEV\",address=\"http://epdkhqr01018.corp.novocorp.net:3076/\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c028a3a5-b178-4437-953b-a5750c6b4f26",
+   "metadata": {},
+   "source": [
+    "# Configuration of methods, nodes, systems and plates"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d19c666b-2b4a-41c9-8ba8-1ea65df3c2df",
+   "metadata": {},
+   "source": [
+    "Using get attributes to: obtain method from list_method, node from list_nodes, system from list_systems within defined node and define the plate type from list_plates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "id": "a920b758-1aaa-4015-a82c-321a7ede0f84",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Method Set Name: Fus_20230504_GSNO_EDTA\n",
+      "Node: Epdkhqr01048\n",
+      "System Configuration Name: 12_5208_HCLASS_BIN\n",
+      "Plate: ANSI-48Vial2mLHolder\n"
+     ]
+    }
+   ],
+   "source": [
+    "list_methods=handler.GetMethodList()\n",
+    "method=list_methods[0]\n",
+    "print(f\"Method Set Name: {method}\")\n",
+    "\n",
+    "list_nodes=handler.GetNodeNames()\n",
+    "node=list_nodes[40]\n",
+    "print(f\"Node: {node}\")\n",
+    "\n",
+    "list_systems=handler.GetSystemNames(node=node)\n",
+    "system=list_systems[0]\n",
+    "print(f\"System Configuration Name: {system}\")\n",
+    "\n",
+    "list_plates=handler.GetPlateTypeNames()[:6]\n",
+    "plate=list_plates[5]\n",
+    "print(f\"Plate: {plate}\")\n",
+    "plates={\"1\": plate, \"2\":plate}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8681ceb9-def5-4ed8-8c23-ee941e3f1b2c",
+   "metadata": {},
+   "source": [
+    "Defining sample position counting logic and varying sample position in sample_list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "id": "c8cd7f19-0e87-4585-b474-e7cbd5816711",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{'Method': 'Fus_20230504_GSNO_EDTA',\n",
+       "  'SamplePos': '1:A,1',\n",
+       "  'SampleName': 'Sample',\n",
+       "  'InjectionVolume': 3},\n",
+       " {'Method': 'Fus_20230504_GSNO_EDTA',\n",
+       "  'SamplePos': '1:A,2',\n",
+       "  'SampleName': 'Sample',\n",
+       "  'InjectionVolume': 3},\n",
+       " {'Method': 'Fus_20230504_GSNO_EDTA',\n",
+       "  'SamplePos': '1:A,3',\n",
+       "  'SampleName': 'Sample',\n",
+       "  'InjectionVolume': 3}]"
+      ]
+     },
+     "execution_count": 74,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#Varying vial position\n",
+    "\n",
+    "sample_set_method_name=\"test_optihplc_1\"\n",
+    "\n",
+    "platelist=[1,2]\n",
+    "rowlist=[i+1 for i in range(8)]\n",
```

Comment:
We can get the number of columns and rows on a plate from Empower Web API. I would like not to try and infer positions from that, but it is a possibility if we really want to.